diff.module

```diff
@@ -177,7 +177,7 @@ function diff_menu_alter(&$callbacks) {
  */
 function diff_admin_paths_alter(&$paths) {
   // By default, treat all diff pages as administrative.
-  if (config_get('diff.settings', 'diff_admin_path . node')) {
+  if (config_get('diff.settings', 'diff_admin_path.node')) {
     $paths['node/*/revisions/view/*/*'] = TRUE;
```

Comment:
In my testing, the default layout is getting used here, not the admin layout if the admin theme is getting used. I believe this is because the menu item defined by `diff_menu()` is `$items['node/%node/revisions/view']`, so that's the path matched by Layout module. To get the right layout being selected to match the theme, I change it to this:

```suggestion
    $paths['node/*/revisions/view'] = TRUE;
```